src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int AMOUNT_INDEX = 1;
+    public static final int OPERATION_TYPE_INDEX = 0;
+    public static final int SUPPLY_DATA_INDEX = 0;
+    public static final int BUY_DATA_INDEX = 1;
+    public static final String COMMA_CHAR = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeReport(toFileName, report);
+    }
+
+    private int[] readFile(String fromFileName) {
```

Comment:
We need to change this method. Method readFile must **only read the file** (similar to writeToFile method). No any calculation. Move calculation to createReport method. Method readFile should return `String` or `List<String>`

Reply:
changed